openedx_events/bridge/avro_attrs_bridge.py

```diff
@@ -269,27 +229,27 @@ def dict_to_attrs(self, data: dict, attrs_cls):
                             raise Exception(
                                 f"Necessary key: {attribute.name} not found in data dict"
                             )
-                    elif attribute.type not in AVRO_TYPE_FOR:
+                    elif attribute.type not in PYTHON_TYPE_TO_AVRO_MAPPING:
                         raise TypeError(
                             f"Unable to deserialize {attribute.type} data, please add extension for custom data type"
                         )
 
-        return attrs_cls(**data)
+        return obj_type(**data)
 
 
 class AvroAttrsBridgeKafkaWrapper(AvroAttrsBridge):
```

Comment:
We are going to want other helpers here that grab the event type and send it to Kafka as well. Maybe to be left until later, but I imagine that this should live with other Kafka code.